app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html

```diff
@@ -49,13 +49,6 @@
 
 @headerNavLinks = {
         <li>
-            <a href="/get-help" id="get-help"
```

Comment:
will we ever need it in the future? Is it safer to comment it out for now?

Reply:
apparently we won't need this in the future so deleting is ok.

Reply:
Done

Reply:
oh ok